Assets/Scripts/Behaviours/World/Cell.cs

```diff
@@ -87,7 +87,16 @@ internal void CreateStaticGeometry ()
 
 			m_insts = new Dictionary<Instance,StaticGeometry> (48 * 1024);
 			foreach (var plcm in placements) {
-				m_insts.Add (plcm, StaticGeometry.Create ());
+                if(DynamicObjectsManager.Instance.IsModelDynamic(plcm.ObjectId))
```

Comment:
Nice job finding the place for this.

Here we have to determine if object is dynamic or not.